astropy/units/quantity.py

```diff
@@ -375,7 +375,7 @@ def __new__(cls, value, unit=None, dtype=None, copy=True, order=None,
                     copy = False  # copy will be made in conversion at end
 
         value = np.array(value, dtype=dtype, copy=copy, order=order,
-                         subok=False, ndmin=ndmin)
+                         subok=True, ndmin=ndmin)
```

Comment:
@mhvk - this is a notable change in existing code. Can you discuss the implications and why it is OK?

Reply:
I had split this off to #11210 as it is an issue that affects `Distribution` as well, but that is still awaiting proper review.